OpenRA.Mods.Cnc/Traits/Buildings/ClonesProducedUnits.cs

```diff
@@ -32,7 +36,7 @@ public class ClonesProducedUnits : ConditionalTrait<ClonesProducedUnitsInfo>, IN
 		readonly Production[] productionTraits;
 
 		public ClonesProducedUnits(ActorInitializer init, ClonesProducedUnitsInfo info)
-			: base(info)
+		   : base(info)
```

Comment:
Please revert this change.

Reply:
thanks @abcdefg30, i've reverted the random extra tab removal